package.json

```
@@ -158,6 +158,7 @@
     "@wpmedia/shared-styles": "file:blocks/shared-styles",
     "@wpmedia/simple-list-block": "file:blocks/simple-list-block",
     "@wpmedia/single-chain-block": "file:blocks/single-chain-block",
+    "@wpmedia/single-column-layout-block@local": "file:blocks/single-column-layout-block",
```

```suggestion
    "@wpmedia/single-column-layout-block": "file:blocks/single-column-layout-block",
```
shouldn't need local for testing

---

updated